### ISSUE TITLE
Can't change TransparentTintBackdrop.TintColor at runtime

### ISSUE THREAD

**Issue by frozax** [dotMorten/WinUIEx#149]
Hello,

I'm trying to change the `TintColor` of a `TransparentTintBackdrop` at runtime, using something similar to:
```
(SystemBackdrop as TransparentTintBackdrop).TintColor = myColor;
```
It's not working, the color is not changed.

I also tried to change the whole backdrop, with something like:
```
SystemBackdrop = new TransparentTintBackdrop(myColor);
```
In this case, it does indeed change the backdrop. However, there is a visible ugly glitch (as if there was one frame without backdrop at all). I also tried to precompute the backdrops to avoid the `new`, but I still get the glitch.

I'm using version 2.3.3.

**Comment by frozax** [comment]
If I change the method `CreateBrush` in `TransparentTintBackdrop` to this:
```
        protected override Windows.UI.Composition.CompositionBrush CreateBrush(Windows.UI.Composition.Compositor compositor)
        {
            brush = WindowManager.Compositor.CreateColorBrush(TintColor);
            return brush;
        }
```

It works.
Hope this helps.

**Comment by dotMorten** [comment]
@frozax Thank you! Your fix was spot on.

**Comment by dotMorten** [comment]
Just a heads up that this fix is in the latest 2.3.4 release
